mcs/class/System/Test/System.Net.NetworkInformation/NetworkInterfaceTest.cs

```diff
@@ -70,6 +77,9 @@ public void FirstInterfaceOperationalStatus ()
 		[Test]
 		public void FirstInterfaceSpeed ()
 		{
+			if (RunningOnUnix)
```

Comment:
In our implementation, we just return a bogus value (), while corefx actually tries to get the value, but the OS returns 0.